### DISCUSSION TITLE
One question about the offline plugin

### DISCUSSION THREAD

**Discussion by icebilly** [squidfunk/mkdocs-material#4003] [Q&A] [answered]
Hi @squidfunk , I want to use the offline "site" package to deliver some doc with our products, but it was inconvenient when navigate to a internal link, as below, users need to click that index.html to open the link. So if I join the sponsor plan, and the insider version will fix this? How it will look like when I use the offline version.
![image](https://user-images.githubusercontent.com/21252620/172378956-07a54554-c26d-4451-b773-c75014d3ee18.png)

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking! You need to set [`use_directory_urls`](https://www.mkdocs.org/user-guide/configuration/#use_directory_urls) to `false` in `mkdocs.yml` 😊 The offline plugin will do this automatically for you and make sure that search works when viewed via `file://`.

**Reply by icebilly** [reply]
Thanks for the help!
